cmake/emu/simics.cmake

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+find_program(
+  SIMICS
+  NAMES simics
+  )
+
+set(SIMICS_SCRIPT_ALTERNATE $ENV{SIMICS_SCRIPT_ALTERNATE})
```

Comment:
`SIMICS_SCRIPT_ALTERNATE` is it for an oot script ? - might its name be more clear, smth. like a `SIMICS_SCRIPT_PATH` ?